tests/disaster-recovery/mdr/test_app_failover_and_relocate.py

```diff
@@ -0,0 +1,226 @@
+import logging
+import pytest
+import time
+
+
+from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.framework import config
+from ocs_ci.ocs.acm.acm import AcmAddClusters
+from ocs_ci.ocs.dr.dr_workload import validate_data_integrity
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import wait_for_nodes_status, get_node_objs
+from ocs_ci.ocs.resources.pod import wait_for_pods_to_be_running
+from ocs_ci.helpers.dr_helpers import (
+    enable_fence,
+    enable_unfence,
+    get_fence_state,
+    failover,
+    relocate,
+    set_current_primary_cluster_context,
+    set_current_secondary_cluster_context,
+    get_current_primary_cluster_name,
+    get_current_secondary_cluster_name,
+    wait_for_all_resources_creation,
+    wait_for_all_resources_deletion,
+    gracefully_reboot_ocp_nodes,
+)
+from ocs_ci.helpers.dr_helpers_ui import (
+    failover_relocate_ui,
+    verify_failover_relocate_status_ui,
+)
+from ocs_ci.utility import version
+
+logger = logging.getLogger(__name__)
+
+polarion_id_primary_up = ""
+polarion_id_primary_down = "OCS-4346"
+
+
+@tier1
+class TestApplicationFailoverAndRelocate:
+    """
+    Test Failover and Relocate actions for a busybox application
```

Comment:
please have design such that multiple apps also can be handled.

Reply:
In `mdr_workload.yaml` we need to pass the repo details, where we can pass such that multiple applications to be created.